storage/innobase/include/ut0new.h

```diff
@@ -663,8 +663,8 @@ class ut_allocator {
 
 		/* e.g. "btr0cur", derived from "/path/to/btr0cur.cc" */
 		char		keyname[FILENAME_MAX];
-		const size_t	len = ut_basename_noext(file, keyname,
-							sizeof(keyname));
+		const size_t	len MY_ATTRIBUTE((unused)) =
```

Comment:
Why? It's used in ut_a below, which should not be compiled away in any config

Reply:
ut_a is defined to be assert(expr) here, which isn't compiled in release builds.

Reply:
I see, this must be innochecksum then